evap/staff/tests/test_views.py

```diff
@@ -485,8 +478,9 @@ def test_access_to_semester_with_archived_results(self):
         # reviewers shouldn't be allowed to access the semester page
         self.app.get('/staff/semester/2', user=reviewer, status=403)
 
-        # managers can access the page
```

Comment:
(here and above) unnecessary run_in_staff_mode in a WebTestStaffMode?

Reply:
This one was needed because the session behaves weird when switching users. But I mitigated it by changing the order of checks here, so this is now fixed.